Assets/MixedRealityToolkit/_Core/Inspectors/MixedRealityConfigurationProfileInspector.cs

```diff
@@ -14,16 +15,24 @@ public class MixedRealityConfigurationProfileInspector : MixedRealityBaseConfigu
     {
         private static readonly GUIContent NewProfileContent = new GUIContent("+", "Create New Profile");
 
+        // Camera properties
         private SerializedProperty enableCameraProfile;
         private SerializedProperty cameraProfile;
+        // Input system properties
         private SerializedProperty enableInputSystem;
         private SerializedProperty inputSystemType;
         private SerializedProperty inputActionsProfile;
         private SerializedProperty enableSpeechCommands;
         private SerializedProperty speechCommandsProfile;
         private SerializedProperty enableControllerProfiles;
         private SerializedProperty controllersProfile;
+        // Boundary system properties
         private SerializedProperty enableBoundarySystem;
+        private SerializedProperty boundaryExperienceScale;
```

Comment:
We should also add a system type for users so choose which boundary class to use. (Similar to the input system type choice.

Reply:
To be sure I understand this feedback, adding this will let users select from alternative implementations (perhaps their own). Am I thinking right?

Reply:
Agreed, Like the InputSystem, the property in the Main MRTK Config Profile needs to use a Systemtype bound to the IMixedRealityBoundarySystem interface.  Then anyone can create their own boundary implementation based off out interface and use it.
Base principle of vNext with any Core component